control-station/src/components/SensorBoxes/Sensors/style.css

```diff
@@ -0,0 +1,25 @@
+#sensorbox{
+    height: 25vh;
+    width: 40%;
```

Comment:
Suggestion [repeating]: using viewport proportions or percentages assumes this content will be displayed in a highly particular layout with specific content dimensions. Consider using a flex or grid-based approached.

Reply:
Claiming okay for now but will want to revisit this.